src/components/comment-input.component.js

```diff
@@ -76,8 +77,14 @@ export class CommentInput extends Component {
     };
   }
 
-  handleSubmit = (body: string): void => {
-    this.props.onSubmitEditing(body);
+  handleSubmitEditing = (body: string): void => {
+    if (Platform.OS === 'android') {
```

Comment:
Yes it tested without the condition, new lines are added but it still "a bug" when I post the comment (nothing is posted)